.tekton/push.yaml

```diff
@@ -64,6 +64,17 @@ spec:
           - name: source
             workspace: workspace
 
+      - name: apply-additional-image-tags
+        params:
+          - name: IMAGE
+            value: quay.io/konflux-ci/appstudio-utils:{{ revision }}
```

Comment:
It would be nice to use the result from the `build-container` task, something like `$(tasks.build-container.results.IMAGE_REF)`.

Reply:
Oh, good thinking. Updated.